app/femr/common/IItemModelMapper.java

```diff
@@ -138,7 +138,7 @@ PatientItem createPatientItem(int id,
      * @param medication
      * @return a new PrescriptionItem or null if processing fails
      */
-    PrescriptionItem createPrescriptionItem(int id, String name, Integer replacementId, String firstName, String lastName,
+    PrescriptionItem createPrescriptionItem(int id, String name, String originalMedicationName, String firstName, String lastName,
```

Comment:
AJ,
Everything seems to work in the pull request. I am concerned that this change doesn't align with the rest of the code. If you were to use this change in ItemModelMapper, then the for loop you wrote in the SearchService (on line 352 - no idea how to link it here) could be completely removed. It would be replaced by sending the string into the stream on line 342 of SearchService. I think you could access the string for mapping more succinctly from the list of IPatientPrescriptions. Something like:

pp.getPatientPrescriptionReplacements().get(0).getOriginalPrescription().getMedication().getName()

It does work either way. I think if we keep the setting of item fields inside the ItemModelMapper it'd be easier to test and maintain later. 

-Kevin